API.md

```diff
@@ -871,7 +871,7 @@ This is the full set of parameters supported by the operation.
 
 When sending large amounts of data or writing in parallel, prefer multipart/form-data over application/x-www-form-urlencoded as the Content-Type.
 
-`parallel-write-handle` and `stream-index` only apply when writing to a replica in parallel. To obtain a parallel-write-handle, see [parallel-write-init](#parallel-write-init).
+`parallel-write-handle` and `stream-index` only apply when writing to a replica in parallel. To obtain a parallel-write-handle, see [parallel_write_init](#parallel_write_init).
```

Comment:
oh, i thought we were moving *to* hyphens everywhere, away from underscores... hmm....

Reply:
Not for the operation names. Parameter names use hyphens as those tend to be phrases. Operation names like `parallel_write_init` use underscores because those are tokens. They are meant to be treated as a whole word. For this particular case, these parameter names are never passed via the URL.

This convention is used throughout the interface and aligns with the info at https://www.google.com/search?q=underscores+in+urls. _(Originally ref'd at  https://github.com/irods/irods_client_http_api/issues/192#issuecomment-1872156185)._